README.md

```diff
@@ -49,7 +50,43 @@ Similarly, [calorimetric.json](https://github.com/building-envelope-data/api/blo
 
 With you web browser, you can search our [wiki](https://github.com/building-envelope-data/api/wiki), the [issues](https://github.com/building-envelope-data/api/issues) and [pull requests](https://github.com/building-envelope-data/api/pulls) and contribute to them.
 
-In order to browse the code conveniently, you should first [clone the repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) and then use a text editor, for example [Visual Studio Code](https://code.visualstudio.com/).
+### In your browser
+
+In order to browse the code conveniently, you can click on `Code` and then `+` to start a new codespace.
```

Comment:
Bei mir ist es notwendig auch das Tab `Cocespaces` zu klicken (zumindest bei der ersten Nutzung). Deswegen schlage ich vor: In order to browse the code conveniently with [Codespaces](https://docs.github.com/en/codespaces/overview), open [building-envelope-data/api](https://github.com/building-envelope-data/api) in your favorite web browser, click on the button "Code" in the top-right corner, select the tab "Codespaces", and on the first usage click on `+` to create a new codespace and on subsequent usages click on the name of an existing codespace.

Reply:
Resolve by [fde4711](https://github.com/building-envelope-data/api/pull/336/commits/fde4711fff3999f6dfea68cac2c6f2fa9a2202b9)